app/gui/qt/lang/sonic-pi_fr.ts

```diff
@@ -652,7 +651,7 @@ Veuillez nous excuser, une erreur critique s&apos;est produite pendant le démar
     <message>
         <location filename="../mainwindow.cpp" line="2038"/>
         <source>Improve readability of code</source>
-        <translation>Amélioration de la lisibilité du code</translation>
+        <translation>Améliore de la lisibilité du code</translation>
```

Comment:
French speaker here (mother tongue), the sentence translates to:
"Améliore la lisibilité du code"
HTH